examples/charts/steps.py

```diff
@@ -1,9 +1,6 @@
 from collections import OrderedDict
 
-import numpy as np
-import pandas as pd
-from bokeh.charts import Chart, Step, Line
-from bokeh.plotting import show,output_file
+from bokeh.charts import Step, show, output_file
```

Comment:
> FWIW. I quite like to be able to do from bokeh.charts import Step, show, instead of needing an extra line just to import show.

I see some value in your opinion, but I also think that the unified approach in `state` have conceptually gains over a "distributed" model where you can use `show` from whatever API...